static/js/notifications.js

```diff
@@ -707,13 +707,6 @@ export function register_click_handlers() {
         e.stopPropagation();
         e.preventDefault();
     });
-    $("#out-of-view-notification").on("click", ".compose_notification_scroll_to_message", (e) => {
-        const message_id = $(e.currentTarget).data("message-id");
-        message_lists.current.select_id(message_id);
-        navigate.scroll_to_selected();
-        e.stopPropagation();
-        e.preventDefault();
-    });
```

Comment:
@alya there's a product question here -- we used to have a "Scroll to message" button in the "Sent! Scroll down to view your message" notice, and it sounds like that got removed at some point -- do we want to remove the remaining bits of its implementation or is would it be useful to have such a button?

(I suspect we may have removed it as part of imagining the "scroll to bottom" button we have in message feeds more generally).

Reply:
moved to #23785